src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -5197,14 +5199,14 @@ VSTGUI::COptionMenu* SurgeGUIEditor::makeZoomMenu(VSTGUI::CRect& menuRect, bool
                        [this, menuRect]() {
                           // FIXME! This won't work on linux
                           char c[256];
-                          snprintf(c, 256, "%d", this->zoomFactor);
+                          snprintf(c, 256, "%f", zoomFactor);
                           promptForMiniEdit(
                               c, "Enter a default zoom level value:", "Set Default Zoom Level",
                               menuRect.getTopLeft(), [this](const std::string& s) {
                                  int newVal = ::atoi(s.c_str());
-                                 Surge::Storage::updateUserDefaultValue(&(this->synth->storage),
+                                 Surge::Storage::updateUserDefaultValue(&(synth->storage),
                                                                         "defaultZoom", newVal);
-                                 this->setZoomFactor(newVal);
+								 resizeWindow(newVal);
```

Comment:
Looks like there's some hard tabs here? Of course when we fix the clang format this will go away but just wanted to point out you have editor set with them. Easy to fix as well. If this is my only comment we can merge and clean up later.